### PR TITLE
Changed from field injection to constructor injection

### PR DESCRIPTION
**Pull request by sabaubogdan**

### CODE REVIEW

**Comment by DavidArt** [comment]
looks ok. you can do the merge

File: src/main/java/xyz/vegaone/easytracking/config/EasyTrackConfig.java [hunk 1]
```diff
@@ -23,5 +23,4 @@ public DataSource dataSource() {
```
Comment (sabaudragos) [inline src/main/java/xyz/vegaone/easytracking/config/EasyTrackConfig.java:28]: For some reason I cannot comment on line 14th, but you have a field injection there. Could you please switch to constructor based injection?

File: src/main/java/xyz/vegaone/easytracking/service/ProjectService.java [hunk 2]
```diff
@@ -19,21 +19,26 @@
 @Service
 public class ProjectService {
```
Comment (sabaudragos) [inline src/main/java/xyz/vegaone/easytracking/service/ProjectService.java:18]: There are some unused imports in this class and some commented code at line 53, could you please remove them?

File: src/test/java/xyz/vegaone/easytracking/project/ProjectServiceTest.java [hunk 3]
```diff
@@ -28,12 +28,16 @@
...
 
-    @Autowired
     private UserStoryService userStoryService;
 
+    @Autowired
+    public ProjectServiceTest(ProjectService projectService, UserStoryService userStoryService) {
+        this.projectService = projectService;
+        this.userStoryService = userStoryService;
+    }
+
```
Comment (sabaudragos) [inline src/test/java/xyz/vegaone/easytracking/project/ProjectServiceTest.java]: This class has some unused imports and commented code as well. Could you please remove them?

**Review by sabaudragos — CHANGES_REQUESTED** [review]

**Review by sabaudragos — APPROVED** [review]

**Pull request merged**